docs/manuals/source/TasksAndConcepts/BareosConsole.rst

```diff
@@ -521,8 +521,9 @@ delete
       delete volume=<volume-name> pool=<pool-name>
       delete JobId=<job-id> JobId=<job-id2> ...
       delete Job JobId=n,m,o-r,t ...
+      delete storage=<storage-name>
 
-   The first form deletes a Pool record from the catalog database. The second form deletes a Volume record from the specified pool in the catalog database. The third form deletes the specified Job record from the catalog database. The last form deletes JobId records for JobIds n, m, o, p, q, r, and t. Where each one of the n,m,... is, of course, a number. That is a "delete jobid" accepts lists and ranges of jobids.
+   The first form deletes a Pool record from the catalog database. The second form deletes a Volume record from the specified pool in the catalog database. The third form deletes the specified Job record from the catalog database. The fourth form deletes JobId records for JobIds n, m, o, p, q, r, and t. Where each one of the n,m,... is, of course, a number. That is a "delete jobid" accepts lists and ranges of jobids. The last form deletes the selected storage from the database, only if it is orphaned, meaning if still persists in the database even though its configuration file has been deleted.
```

Comment:
`... its configuration has been removed and there are no volumes associated with it anymore`